support-frontend/assets/components/ticker/contributionTicker.tsx

```diff
@@ -158,7 +158,7 @@ export default class ContributionTicker extends Component<
 			);
 		}
 
-		if (this.state.goalReached && this.props.tickerEndType === 'unlimited') {
+		if (this.props.tickerEndType === 'unlimited') {
```

Comment:
We don't need the `goalReached` check as we've already checked `!goalReached` above and returned from the function if so